clang/test/Driver/clang-offload-bundler.c

```diff
@@ -5,7 +5,7 @@
 // Generate all the types of files we can bundle.
 //
 // generate the emulated fat object:
-// RUN: %clangxx -O0 -target powerpc64le-ibm-linux-gnu -DEMULATE_FAT_OBJ %s -c -o %s.o
+// RUN: %clangxx -O0 -target powerpc64le-ibm-linux-gnu -DEMULATE_FAT_OBJ %s -c -o %t.o.2
```

Comment:
@mdtoguchi, any suggestions on better name for this temporary file?

I cannot just use `%t.o` because it conflicts with `%t.o` used on line 15 and this causes test failure

Reply:
@AlexeySachkov, you can just use %t_2.o or something similar if you want to retain the .o extension.